src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './Tabs.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+class Tabs extends React.Component {
+  changeStatusHandler = (id) => {
+    const { idSelectedTab, onChangeSelTabs } = this.props;
+
+    if (idSelectedTab !== id) {
+      onChangeSelTabs(id);
+    }
+  }
+
+  render() {
+    const { tabs, idSelectedTab } = this.props;
+    const selectedTab = tabs.find(tab => tab.id === idSelectedTab);
```

Comment:
We already have such string in the component App. It would be better if we find selectedTab only once